tools/depends/target/python27/Makefile

```diff
@@ -14,6 +14,7 @@ endif
 ifeq ($(OS),ios)
   HOSTPLATFORM=_PYTHON_HOST_PLATFORM="darwin"
   LINK_ICONV=-framework CoreFoundation -liconv
+  CFLAGS+=-DDISABLE_PROCESS_CREATION
```

Comment:
doesn't fly - i think i simply change the patch to disable fork and co and apply it only for appletvos getting rid of the define ...